### PR TITLE
Add FitzBundle contrib recipe

### PR DESCRIPTION
**Pull request by hugosoltys**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by hugosoltys** [comment]
@Nyholm can you take a look? I was told to contact you.

File: hugosoltys/fitz-bundle/1.0/config/packages/test/fitz.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fitz:
```
Comment (Nyholm) [inline hugosoltys/fitz-bundle/1.0/config/packages/test/fitz.yaml]: One could do:
```
imports:
    - { resource: ../dev/hugosoltys_fitz.yaml }
```

But that is up to you

**Comment by hugosoltys** [comment]
@Nyholm I did what you asked me to do, but now the CI seems stucked on `Pending — Validating the pull request`. Any idea ?

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Pull request merged**